app/views/admin/document_collection_group_memberships/index.html.erb

```diff
@@ -19,18 +19,40 @@
       items: secondary_navigation_tabs_items(@group, request.path),
     } %>
 
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full app-view-document-collection-group-memberships-index__heading">
-        <%= render "govuk_publishing_components/components/heading", {
-          text: "Documents",
-          margin_bottom: 6,
-        } %>
+    <% if @group.memberships.many? %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full app-view-document-collection-group-memberships-index__heading ">
```

Comment:
These links should really be marked up as list items, based on how it's done in the Summary List component - [With edit and delete on section](https://components.publishing.service.gov.uk/component-guide/summary_list/with_edit_and_delete_on_section).

Reply:
@nnagewad understand where David is coming from - however there is a slightly different design between current implementation and the summary list edit and delete on section designs. Namely the horizontal line separating the two links. Happy to change, look forward to hearing your feedback on this! 

See screenshot of edit and delete section via summary list component:
<img width="168" alt="Screenshot 2023-10-05 at 15 20 19" src="https://github.com/alphagov/whitehall/assets/138006225/e17da653-ec58-491d-9ff5-70bec02fc587">

Reply:
The most important thing here is to use the appropriate semantic markup for these elements, whilst also maintaining consistency. The styling should follow that. So I think if we follow the markup and styling in the Summary List component (see the link above) that should render this correctly both semantically and stylistally. Happy to get together on this if necessary.

Reply:
Hi @RodneyJohnsonGDS could we just make sure this issue is finalised before merging?

Reply:
I think we can change the existing markup to something like below then adapt the CSS to match (though I still think it would be even simpler to use the Summary List component here in place of the Table and the heading)

```
ul class="govuk-list">
  <li>
    <%= link_to "Add document", admin_document_collection_group_search_options_path(@collection, @group), class: "govuk-link" %>
  </li>

  <li>
    <%= link_to "Reorder document", reorder_admin_document_collection_group_document_collection_group_memberships_path(@collection, @group), class: "govuk-link" %>
  </li>
</ul>
```